### ISSUE TITLE
User can cancel an order

### ISSUE THREAD

**Issue by stevebrownlee** [pushy-donkeys/Bangazon-Site#9]
**Given** the user is authenticated
**When** the user is viewing their order
**Then** there should be a button labeled *Cancel Order* which will remove all products from the order and delete the order

**Comment by therealrmac** [comment]
does this affect the Quantity available?

**Comment by megducharme** [comment]
@therealrmac no, this will not affect the quantity. The only time a quantity will be decremented is when a order is completed. An order cannot be deleted unless it's open (not completed).
